src/main/java/core/basesyntax/service/ReportBuilder.java

```diff
@@ -0,0 +1,8 @@
+package core.basesyntax.service;
+
+import core.basesyntax.model.FruitTransaction;
+import java.util.List;
+
+public interface ReportBuilder {
+    List<String> getReport(List<FruitTransaction> transactions);
```

Comment:
```suggestion
    String getReport(Map<String, Integer> data);
```
create storage to store fruit information, get that data and pass here in this method as a parameter